AdminPortal/src/styles/Login.module.css

```diff
@@ -0,0 +1,147 @@
+.outer {
+    display: flex;
+    height: 600px;
```

Comment:
Do you have an automatic code formatter installed? If not, then I would recommend using Prettier. CSS files should also have 2-space indents

Reply:
I do have it installed, however I had tab indentation set to tab of width 4, have just changed that to 2 now!